README.md

```diff
@@ -4,39 +4,75 @@ Validation of OpenShift Pipeline releases
 
 ## Setup
 
-### Install goconvey
+### Prerequisite 
+
+#### OCP cluster (4.2.*)
+
+#### Installing `tkn`
+Download the latest binary executable for your operating system:
+
+[tkn install](https://github.com/tektoncd/cli/blob/master/README.md#installing-tkn)
+
+#### Installing `oc`
+Download `oc` latest binary executable for your operating system
+
+
+### Getting Ginkgo
 
 ```
-go get github.com/smartystreets/goconvey
+go get github.com/onsi/ginkgo/ginkgo
+go get github.com/onsi/gomega/...
 ```
 
-ref: https://github.com/smartystreets/goconvey#installation
+ref: http://onsi.github.io/ginkgo/#getting-ginkgo
 
 ## Running tests
 
 ```
-goconvey
+ginkgo -v ./spec/... --timeout 10m
```

Comment:
What is this timeout good for? My test was running over 60 min when I specified a timeout of 20m.

Reply:
60 mins? that's strange It should stop execution by 20mins at any cost, even if tests didn't pass

Reply:
It's because of ```APITimeOut``` we have provided in pkg/config/config.go